src/elements/content-sidebar/AddTaskButton.js

```diff
@@ -67,36 +53,19 @@ class AddTaskButton extends React.Component<Props, State> {
         const { isDisabled, feedbackUrl, ...passThrough } = this.props;
         const { isTaskFormOpen, taskType, error } = this.state;
 
-        // CSS selector for first form element
-        // Note: Modal throws an error if this fails to find an element!
-        const focusTargetSelector = '.task-modal input';
         return (
             <React.Fragment>
                 <AddTaskMenu isDisabled={isDisabled} onMenuItemClick={this.handleClickMenuItem} />
-                <Modal
-                    className="be-modal task-modal"
-                    data-testid="create-task-modal"
-                    focusElementSelector={focusTargetSelector}
-                    isOpen={isTaskFormOpen}
-                    onRequestClose={this.handleModalClose}
-                    title={
-                        <React.Fragment>
-                            <FormattedMessage {...this.getMessageForModalTitle(taskType)} />
-                            <BetaFeedbackBadge tooltip formUrl={feedbackUrl} />
-                        </React.Fragment>
-                    }
-                >
-                    <div className="be">
-                        <TaskForm
-                            {...passThrough}
-                            error={error}
-                            onCancel={this.handleModalClose}
-                            onCreateSuccess={this.handleCreateSuccess}
-                            onCreateError={this.handleCreateError}
-                            taskType={taskType}
-                        />
-                    </div>
-                </Modal>
+                <TaskModal
+                    error={error}
+                    feedbackUrl={feedbackUrl}
+                    handleCreateError={this.handleCreateError}
+                    handleCreateSuccess={this.handleCreateSuccess}
+                    handleModalClose={this.handleModalClose}
+                    isTaskFormOpen={isTaskFormOpen}
+                    passThrough={passThrough}
```

Comment:
How about changing this to "taskFormProps" or something since it is now a public interface rather than a private variable?

Reply:
Or continue the pattern and {..rest} the props that are not recognized?

Reply:
IRL:

- rename to `taskFormProps`, add to both AddTaskButton and TaskModal
- export type from TaskForm to converge on source of truth for the form props